recipe/meta.yaml

```diff
@@ -37,13 +30,18 @@ requirements:
     - shapely >=2.0.0,<3.0.0
 
 test:
-  # imports:
-  #   - lonboard
+  imports:
+    - lonboard
   commands:
     - pip check
   requires:
     - pip
 
+outputs:
+  - name: lonboard
+    files:
+      - lonboard
+
```

Comment:
These lines are probably not needed.

Reply:
I was trying to figure out how to get it to include the `static` files in the `lonboard/static` directory. In theory we shouldn't even need to `pip install`, we only want to "reexport" the sources that the pypi package already created.

But I don't know Conda well enough to figure out how to do this. Maybe one of the watchers of this repo knows how

Reply:
Maybe add a `MANIFEST.in` file in the lonboard repo can help reslove the issue. This can make sure the `static` folder is copied over when using conda install?

```text
recursive-include lonboard/static *
```

See this example: 
https://github.com/gee-community/geemap/blob/master/MANIFEST.in

Reply:
Another reference: https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#controlling-files-in-the-distribution

Reply:
I can try that; I'm currently defining that inside the `pyproject.toml`: https://github.com/developmentseed/lonboard/blob/a2b096c7e2833fca80cdcf6eeddbe9a8ef2c4f17/pyproject.toml#L9

Reply:
Somehow the conda install does not copy the `static` over. I am not sure why. Probably conda handles the installation differently.

Reply:
> Somehow the conda install does not copy the `static` over. 

Right. I think this is the only problem we have; otherwise it should just work

Reply:
Seems relevant to this issue. The difference is that lonboard using `pyproject.toml` rather than `setup.py`

https://stackoverflow.com/questions/47935680/conda-build-is-omitting-data-file-sub-directories-even-though-setup-is-includin